osc-server/src/main/java/org/osc/core/broker/service/tasks/conformance/securitygroup/CreateMgrSecurityGroupTask.java

```diff
@@ -69,7 +72,7 @@ public void executeTransaction(EntityManager em) throws Exception {
             String iscId = this.sg.getId().toString();
             String mgrEndpointGroupId = mgrApi.createSecurityGroup(this.sg.getName(), iscId,
                     getSecurityGroupMemberListElement(this.sg));
-            this.sg.setMgrId(mgrEndpointGroupId);
+            this.sgi.setMgrSecurityGroupId(mgrEndpointGroupId);
```

Comment:
This seems odd, shouldnt this loop through all the SGIs in the SG and set the mgrendpointgroupid instead?

Reply:
SGI is per manager per SG so this should be fine